doc/source/getting-started/administration.rst

```diff
@@ -0,0 +1,161 @@
+Project approval and public release
+===================================
+Most of the projects in PyAnsys expose the functionality of ANSYS, Inc.
+products. Due to intellectual property reasons, the public release of a PyAnsys
+library must undergo through a project approval process.
+
+
+Approval process
+----------------
+The approval process is divided into three different parts:
+
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`file-badge` Managerial
+       :link: managerial
+       :link-type: ref
+
+        Verifies that direct managers, GM/VP, CT or CTO are aware about the project and approve it.
+
+    .. grid-item-card:: :octicon:`law` Legal
+       :link: legal
+       :link-type: ref
+
+        Verifies that the project ships with the right legal framework to protect ANSYS, Inc. intellectual property.
```

Comment:
Again, I question why we don't use "... to protect Ansys's intellectual property."

Reply:
Thanks for pointing this out, Kathy.